src/scripts/underground/UndergroundUpgrade.ts

```diff
@@ -53,3 +53,10 @@ Underground.upgradeList.push(
         GameHelper.createArray(0, 2, 1)
     )
 );
+
+Underground.upgradeList.push(
+    new UndergroundUpgrade(Underground.Upgrades.Bomb_Efficiency, 'Bomb Efficiency', 5,
+        AmountFactory.createArray(GameHelper.createArray(50, 500, 50), GameConstants.Currency.diamond),
+        GameHelper.createArray(0, 10, 2)
```

Comment:
I'm honestly not sure if the bonuses are too cheap/expensive as these are just the first numbers that I tried.
Do you think increments of 1 would work out better?

But the arrays make sense to me now thanks.
_(I probably should have just run the array constructor individually to figure it out)_
```suggestion
        AmountFactory.createArray(GameHelper.createArray(50, 250, 50), GameConstants.Currency.diamond),
        GameHelper.createArray(0, 10, 2)
```

Reply:
The total cost to upgrade does seem a little cheaper than the others, except daily deals, but I think it is ok like this.